pom.xml

```diff
@@ -48,27 +43,27 @@
       <artifactId>xercesImpl</artifactId>
       <version>2.11.0</version>
     </dependency>
-    <dependency><!-- 2.4 is required. Avoid transitive import derived issues -->
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.4</version>
-      <scope>provided</scope>
-    </dependency>
```

Comment:
I couldn't find any problems with the build or downstream plugins here… but maybe I'm missing something.